wagtail/project_template/requirements.txt

```
@@ -1,2 +1,2 @@
-Django>=5.2,<5.3
+Django>=5.2,<6.1
```

```suggestion
Django>=6.0,<6.1
```

(While Wagtail itself has a wider range of compatible releases, it's better for the requirements.txt in people's projects to unambiguously specify one minor release, so that they don't end up with different versions across development / staging / production environments.)